staging/src/github.com/openshift/oc/pkg/helpers/describe/describer.go

```diff
@@ -719,7 +719,17 @@ func DescribeImage(image *imagev1.Image, imageName string) (string, error) {
 		formatString(out, "Image Created", fmt.Sprintf("%s ago", formatRelativeTime(dockerImage.Created.Time)))
 		formatString(out, "Author", dockerImage.Author)
 		formatString(out, "Arch", dockerImage.Architecture)
-		describeDockerImage(out, &dockerImage.ContainerConfig)
+
+		if dockerImage.Config != nil {
+			// Config is the configuration of the container received from the client.
+			// In most cases this field is always set for images.
+			describeDockerImage(out, dockerImage.Config)
+		} else {
+			// ContainerConfig is the configuration of the container that is committed into the image.
+			// In some cases, when the dockerImage.Config is not set (legacy docker versions), fallback
+			// to describe ContainerConfig.
+			describeDockerImage(out, &dockerImage.ContainerConfig)
+		}
```

Comment:
might worth checking `dockerImage.ContainerConfig` for nil as well and handling the case with error

Reply:
that is not a pointer, just struct which is empty for only @smarterclayton release image it looks like... @smarterclayton what you did to that image? ;-)

Reply:
@smarterclayton I mean `oc get isimage -n ocp release@sha256:1510013b577e7a7b08d01c8c52ae3c2680a41e77058afd8d0aa95278c0e5651e -o json` give me:

```
            "ContainerConfig": {},
            "Created": "2019-07-02T10:22:43Z",
            "DockerVersion": "1.13.1",
```

Where as the same command against ruby:2.0 image gives me both fields populated.

Reply:
oh, I overlooked that, disregard